src/illuminator/models/construct_model.py

```diff
@@ -0,0 +1,59 @@
+from abc import ABC, abstractmethod
```

Comment:
@jgrguric96 This is my code for an abstract class so far.

Reply:
@jgrguric96 This changes have been merge to the Dev branch. Please update your dev branch to avoid conflicts